src/components/PlantCardInfo/PlantCardInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FunctionComponent, SVGAttributes } from 'react';
+import { Text } from 'react-native';
+import { useStyles } from 'react-native-unistyles';
+import { styleSheet } from './PlantCardInfo.style';
+import { View } from '../Themed';
+
+type PlantCardInfoProps = {
+	title: string;
+	SvgIcon: FunctionComponent<SVGAttributes<SVGElement>>;
+};
+
+export const PlantCardInfo = ({ title, SvgIcon }: PlantCardInfoProps) => {
+	const { styles } = useStyles(styleSheet);
+
+	return (
+		<View style={styles.container}>
+			<SvgIcon width={16} height={16} />
```

Comment:
I think app sizes should also come from theme, rather a random magic numbers. 
HINT: `useStyles` hook should return `theme` object together with` styles`